plugins/Scheduler/plugin.py

```diff
@@ -90,7 +90,9 @@ def _restoreEvents(self, irc):
                         # though we'll never know for sure, because other
                         # plugins can schedule stuff, too.
                         n = int(name)
-                    self._add(network, event['msg'], event['time'], event['command'], n)
+                    isreminder = event.get('isreminder')
```

Comment:
It's equivalent, but I'd like to use `isreminder = event.get('isreminder', False)`, so a missing value returns `False` instead of `None`.

Also, please add a comment explaining that we need `.get()` instead of `[]` older bots didn't have it set